modules/search/instant-search/components/jetpack-colophon.jsx

```diff
@@ -34,11 +34,16 @@ const logoPath = (
 );
 const logoSize = 12;
 
-const JetpackColophon = () => {
+const JetpackColophon = props => {
+	const locale_prefix = props.locale.split( '-', 1 )[ 0 ];
+	const url =
+		locale_prefix && locale_prefix !== 'en'
```

Comment:
Is this distinction necessary? It looks like `en.jetpack.com/search` works swimmingly.

Reply:
Ya, I wasn't going to do it, but since I figured we should handle the case where `local_prefix` isn't set then it seemed worthwhile to also have the link go to the right place rather than relying on the redirect. The more important piece to me wasn't whether the link worked but that when hovering over the link I see en.jetpack.com rather than jetpack.com

Reply:
> when hovering over the link I see en.jetpack.com rather than jetpack.com

Wouldn't this conditional make is so that `en.jetpack.com` is always replaced by `jetpack.com`?

> it seemed worthwhile to also have the link go to the right place rather than relying on the redirect

Gotcha 👍

Reply:
> Wouldn't this conditional make is so that en.jetpack.com is always replaced by jetpack.com?

Sorry, yes, I reversed that.